tests/gnrc_sixlowpan/main.c

```diff
@@ -118,7 +118,7 @@ static void _init_interface(void)
     addr.u8[1] = 0x01;
     addr.u8[15] = 0x01;
 
-    xtimer_usleep(500); /* wait for thread to start */
+    ztimer_sleep(ZTIMER_USEC, 500); /* wait for thread to start */
```

Comment:
Could also be 
```suggestion
    ztimer_sleep(ZTIMER_MSEC, 1); /* wait for thread to start */
```

IIRC there is no specific reason why it is 500us.